benchmarks/solcx/compositional_fields/solcx_particles.prm

```diff
@@ -0,0 +1,133 @@
+# A description of the SolCX benchmark using active particles
```

Comment:
Why is there a solcx_particles file in the compositional_fields folder? Did this happen by accident?